.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+.idea
```

Comment:
You should add these files to your global ignores rather than committing them to this repo.